qiita_db/util.py

```diff
@@ -863,7 +863,7 @@ def convert_to_id(value, table, text_col=None):
                                      % (value, table))
         # If there was a result it was a single row and and single value,
         # hence access to [0][0]
-        return _id[0][0]
+        return int(_id[0][0])
```

Comment:
The docstring says it returns an `int`, but that was not actually always the case. I thought this was the easier/better change (as opposed to changing the docstring to say that it returns `object`, or something to that effect).